assets/js/components/read-more/index.js

```diff
@@ -32,7 +41,21 @@ class ReadMore extends Component {
 
 	componentDidMount() {
 		if ( this.props.children ) {
-			this.lineHeight = this.element.current.clientHeight + 1;
+			const { maxLines } = this.props;
+			const lineHeight = this.reviewSummary.current.clientHeight + 1;
+			const reviewHeight = this.reviewContent.current.clientHeight + 1;
+			const maxHeight = ( lineHeight * maxLines ) + 1;
+
+			this.setState( {
+				maxHeight: maxHeight,
+				isExpanded: false,
+				isClamped: reviewHeight > maxHeight,
+			} );
+		}
```

Comment:
I might be missing something, but couldn't we call `clampLines()` directly inside `componentDidMount()` when `reviewHeight > maxHeight`? This way we can get rid of `componentDidUpdate()`, I think.